flow/Knobs.cpp

```diff
@@ -284,6 +284,7 @@ void FlowKnobs::initialize(Randomize randomize, IsSimulated isSimulated) {
 	init( RESTCLIENT_MAX_CONNECTION_LIFE,                      120 );
 	init( RESTCLIENT_REQUEST_TRIES,                             10 );
 	init( RESTCLIENT_REQUEST_TIMEOUT_SEC,                      120 );
+
```

Comment:
Can you remove this line?

Reply:
Thanks for pointing out. Shall remove it.